.github/workflows/lint.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: norio-nomura/action-swiftlint@3.2.1
         with:
           args: --strict
-      - name: GitHub Action for SwiftLint (Only files changed in the PR)
+      - name:GitHub Action for SwiftLint (Only files changed in the PR)
```

Comment:
What is the benefit of this change?

Reply:
Since there is no reason for this change we should close this?

Reply:
Yes.